pkg/node/resources.go

```diff
@@ -29,7 +29,7 @@ var InstanceCores = map[string]int{
 	"c3.xlarge":     4,
 	"c3.2xlarge":    8,
 	"c3.4xlarge":    16,
-	"c3.8xlarge":    32,
```

Comment:
I think that 32 is the correct number of cores: https://ec2instances.info/?selected=c3.8xlarge